### PR TITLE
lower onnx max op to torch aten maximum op

### PR DESCRIPTION
**Pull request by aldesilv**
lower onnx min op to torch aten minimum op

### CODE REVIEW

File: include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h [hunk 1]
```diff
@@ -54,6 +54,13 @@ struct OpBinder {
     return success();
   }
 
+  ParseResult tensorOperandsList( llvm::SmallVector<Value, 4> &values) {
+    for (int i = 0; i < op->getNumOperands(); i++) {
+      values.push_back(op->getOperand(i));
+    }
+    return success();
+  }
+
```
Comment (vivekkhandelwal1) [inline include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h:63]: Something similar is done here: https://github.com/llvm/torch-mlir/pull/2601/files#diff-473c414724482953c052f65a2f0eb1f8387f62ba9793ac04474437c0e9dd8c2dR57-R59

File: test/Conversion/TorchOnnxToTorch/simple_ops_g_to_p.mlir [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// RUN: torch-mlir-opt <%s -convert-torch-onnx-to-torch | FileCheck %s
+
+// CHECK-LABEL: func.func @test_max_example
+  func.func @test_max_example(%arg0: !torch.vtensor<[3],f32>, %arg1: !torch.vtensor<[3],f32>, %arg2: !torch.vtensor<[3],f32>) -> !torch.vtensor<[3],f32> attributes {torch.onnx_meta.ir_version = 7 : si64, torch.onnx_meta.opset_version = 13 : si64, torch.onnx_meta.producer_name = "backend-test", torch.onnx_meta.producer_version = ""} {
+    %0 = torch.operator "onnx.Max"(%arg0, %arg1, %arg2) : (!torch.vtensor<[3],f32>, !torch.vtensor<[3],f32>, !torch.vtensor<[3],f32>) -> !torch.vtensor<[3],f32>
```
Comment (vivekkhandelwal1) [inline test/Conversion/TorchOnnxToTorch/simple_ops_g_to_p.mlir:5]: Add a check for the newly formed op. Please do the same for all the tests.

**Review by vivekkhandelwal1 — CHANGES_REQUESTED** [review]

**Review by stellaraccident — COMMENTED** [review]
Lgtm overall. Vivek can have the final say and merge.

File: include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h [hunk 3]
```diff
@@ -62,6 +62,13 @@ struct OpBinder {
       return failure();
     return success();
   }
+  
+  ParseResult tensorOperandsList( llvm::SmallVector<Value, 4> &values) {
```
Comment (vivekkhandelwal1) [inline include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h:66]: Use `SmallVectorImpl` here. Also, no need to pass the size.

**Review by vivekkhandelwal1 — CHANGES_REQUESTED** [review]

File: include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h (hunk 1 above)
Comment (aldesilv) [reply include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h]: Ok

**Comment by stellaraccident** [comment]
Can you please resolve conflicts and then we can land?

**Pull request merged**